docs/Module-Review/MPEs/CZ4071 Network Science.md

```diff
@@ -0,0 +1,31 @@
+---
+id: cz4071-network-science
+slug: cz4071-network-science
+sidebar_position: 21
+title: CZ4071 Network Science
+description: Learning network properties not limited to: metrics, querying, analytics, ...
+keywords: [network, science, ntu, scse, module, course]
```

Comment:
Gettinga YAML parse error that causes a build failure on this line due to the `:`. Perhaps we can change this to just `Learning about network properties` or something simple like that